functions/Show-DbaServerFileSystem.ps1

```diff
@@ -221,6 +221,6 @@ Shows a GUI and uses SQL credentials to log into the SQL Server. Returns a strin
 			return $drive
 		}
 		
-		Test-DbaDeprecation -DeprecatedOn "1.0.0" -Silent:$false -Alias Show-SqlServerFileSystem
+		Test-DbaDeprecation -DeprecatedOn "1.0.0" -Silent:$false -Alias Show-SqlInstanceFileSystem
```

Comment:
There's no reference to Show-SqlInstanceFileSystem, this change should be reverted or the command should be renamed.